core/app/codec.go

```diff
@@ -1,21 +1,24 @@
 package app
 
 import (
-	"github.com/cosmos/gogoproto/jsonpb"
-	gogoproto "github.com/cosmos/gogoproto/proto"
+	gogoproto "cosmossdk.io/core/transaction"
```

Comment:
We should either not alias this and just call it `transaction.Msg` or have a `core/proto` package so this can simply be `proto.Message`

Reply:
I'll remove the alias. It was mainly for testing but didn't clean it up